packages/service/ni_measurement_plugin_sdk_service/measurement/service.py

```diff
@@ -411,12 +411,15 @@ def configuration(
                 "DataType.PinArray1D is deprecated. Use DataType.IOResourceArray1D instead.",
                 DeprecationWarning,
             )
+        # TODO: Check if display_name is used anywhere
+        # if not then validate display_name here
+        field_name = parameter_metadata.validate_display_name(display_name)
```

Comment:
@dixonjoel @LazeringDeath Are you intentionally breaking compatibility with existing measurements that don't follow Protobuf naming style? Are you doing this for LabVIEW measurement services and the InstrumentStudio/TestStand plugins too?

If you are preserving compatibility, you can't merge display_name and field_name. The display_name field has to match the `.measui` file, and replacing spaces with underscores will make it not match.

Most of the Python examples use control/indicator labels with Protobuf style names and captions with English names (including spaces, parens, etc.), but ye olde SampleMeasurement mixes these up: https://github.com/ni/measurement-plugin-python/blob/main/examples/sample_measurement/SampleMeasurement.measui

Reply:
I see, I suggested to merge them together after checking the display_name isn't being used anywhere else which I thought was true but I got confused from the 2 display names issued [here](https://github.com/ni/measurement-plugin-python/issues/336). I'll go revert that.